### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Disk_UploadVHD_ScreenReader: Screen reader announces the browse button as "No file Selected" which is incorrect. 

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#2410]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #NVDA; #JAWS; #Win32; #CELAAssessID_CosmosDB-FY20-Q2; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.11.1
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Disks" of the selected subscription.
5. Navigate to "A11ytest" and click on it.
6. Navigate to upload button and click on it.
7. "Upload VHD" Popup gets opened.
8. Start NVDA and using Tab key and check whether the NVDA reads the browse button as "No file Selected".

**​Actual:​**
NVDA announces the browse button as "No file Selected" which is incorrect.

Same issue is Observed in **JAWS** also.

**Expected:**
NVDA should announce as browse button.

**UserImpact:**
Screen reader user unable to know the actual behavior of the button.

**Comment by MRayermannMSFT** [comment]
Related: https://github.com/microsoft/AzureStorageExplorer/issues/2068

Which already was merged to master. So fixed.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by vinodbk197** [comment]
Verified this issue in Microsoft Azure Tools Storage Explorer Application Version: 1.12.0, this issue is no longer reproduced. Hence Closing the bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
